wire-ios-transport/Source/Requests/RequestAvailableNotification.swift

```diff
@@ -31,15 +33,15 @@ private let RequestsAvailableNotificationName = "RequestAvailableNotification"
 @objc(ZMRequestAvailableNotification) public class RequestAvailableNotification: NSObject {
 
     @objc public static func notifyNewRequestsAvailable(_ sender: NSObjectProtocol?) {
-        NotificationCenter.default.post(name: Notification.Name(rawValue: RequestsAvailableNotificationName), object: nil)
+        NotificationCenter.default.post(name: .requestAvailableNotification, object: nil)
```

Comment:
should we pass the sender here

Reply:
I don't see the sender being used for anything

Reply:
that's another issue, but either we keep the parameter and at least pass the info especially posting, or we should remove the param all together

Reply:
Right we should but maybe not in this PR, I just exposed a notification name. I can't refactor all the code I touch.